Forms/MainForm.cs

```diff
@@ -1117,5 +1378,82 @@ private void TogglePatchSettings(bool v)
                 tPatch.Text = null;
             }
         }
+
+
+
+        private void WritePreset(string filename)
+        {
+            StreamWriter PresetFile = new StreamWriter(File.Open(filename + PresetExt, FileMode.Create));
+            PresetFile.WriteLine("MMR Preset File [" + AssemblyVersion + "]");
+            PresetFile.WriteLine(tSString.Text);
+            PresetFile.WriteLine(tCustomItemList.Text);
+            PresetFile.WriteLine(tStartingItemList.Text);
+
+            if(_settings.LogicMode == LogicMode.UserLogic)
+            {
+                if(_settings.UserLogicFileName != null && File.Exists(_settings.UserLogicFileName))
```

Comment:
The logic should not be saved to the settings file by default. Frequent users of user logic update their logic file often, and they'll be forced to constantly change their Logic Mode from Preset to Custom and re-select their logic file. Would be useful to have an option to save logic to the settings file in case someone wants to share their settings with a friend.

Reply:
All you would have to d is change to casual logic before saving the logic file in order to avoid saving custom logic to the cfg.

Reply:
And then they'll have to change the settings to user logic and select a logic file every time they load the randomizer and want to use their user logic that they use all the time?